views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,82 @@
             </div>
         {% endif %}
         {% if upgradeInfo.version_num is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Version number:'|trans }}</label>
-                <span class="version">{{ upgradeInfo.version_num }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Version number:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <span class="version">{{ upgradeInfo.version_num }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.link is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'URL:'|trans }}</label>
-                <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'URL:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+                </div>    
             </div>
         {% endif %}
         {% if upgradeInfo.md5 is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
-                <span class="md5">{{ upgradeInfo.md5 }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <span class="md5">{{ upgradeInfo.md5 }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.changelog is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Changelog:'|trans }}</label>
-                <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Changelog:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">  
+                    <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+                </div>
             </div>
         {% endif %}
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'PHP compatibility:'|trans }}</label>
+            </div>
+            <div class="col-xs-9 col-md-11">
+                <span class="phpcompatibility">{{ 'Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.
```

Comment:
> Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.

Well, actually, no. You can upgrade from a version A to a version B that do not share PHP compatibility. But's it's harder 😅  .

I think we'll require the help of @Julievrz to find the right ways to explain this 😉

Reply:
I follow https://www.figma.com/file/GpRsUUCHcdhJPAaz7Eqzes/Autoupgrade?node-id=497%3A11710 but i agree

Reply:
> Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.

Well, actually, no. You can upgrade from a version A to a version B that do not share PHP compatibility. But's it's harder 😅 .

In the next version, the goal is to not allow the user to upgrade if his PHP version isn't compatible with both his current PS version and the one to which he wants to upgrade.
The message to display above the compatibility table is the one defined in the mockup ([see this issue](https://github.com/PrestaShop/PrestaShop/issues/25659)): "Your PHP version must be compatible with both your current PrestaShop version and the one you want to upgrade to.